components/inputs/test/input-number.test.js

```diff
@@ -455,6 +455,7 @@ describe('d2l-input-number', () => {
 			const elem = await fixture(minMaxFixture);
 			setTimeout(() => setInnerInputValue(elem, '0'));
 			await oneEvent(elem, 'change');
+			await elem.updateComplete;
 			expect(elem.invalid).to.be.true;
```

Comment:
Would it be possible to add a test that would have originally failed before this fix but now passes? Just so we don't accidentally re-introduce it.